tests/support/util.tcl

```diff
@@ -1162,3 +1162,18 @@ proc generate_largevalue_test_array {} {
     set largevalue(quicklist) [string repeat "x" 8192]
     return [array get largevalue]
 }
+
+proc bp {{s {}}} {
```

Comment:
Maybe add a brief comment here about what this is, since bp is not a very useful description and neither is s as an argument.